core/services/job/orm.go

```diff
@@ -436,6 +462,7 @@ func (o *orm) FindJob(id int32) (Job, error) {
 	var job Job
 	err := postgres.GormTransactionWithDefaultContext(o.db, func(tx *gorm.DB) error {
 		err := PreloadAllJobTypes(tx).
+			Preload("JobSpecErrors").
```

Comment:
JobSpecErrors previously had a presenter but we forgot to preload them. I think as a pattern we should pass in preloads into the ORM function so that it's clear what exactly we're preloading.